### ISSUE TITLE
NTR: Francisella tularensis antibody titer high in blood (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8094]
New term request Francisella tularensis antibody titer high in blood (seidel_190313154731): Tularemia; antibodies  UMLS: 2748094

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000740
- label: Anti-Francisella tularensis antibody positivity
- parents: [Positive bloodstream antibacterial antibody positivity (HP:0430061)](https://hpo.jax.org/app/browse/term/HP:0430061)
- pmid: 
- def: The presence of antibodies in the blood circulation that react against a component of Francisella tularensis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8094
- synonyms:
